### DISCUSSION TITLE
Want to contribute

### DISCUSSION THREAD

**Discussion by LabibAhmedSiddique** [rochi88/bdshare#8] [Ideas]
I've had my eye on your repo for quite a while now and your tool has been extremely helpful. Apparently, I have been working on scraping dse data for numerous projects. I have made a basic understanding about the DSE website structure and scrape from  dse company page, and I would love to use my knowledge to contribute to the community , I'm open to suggestions for any additional features you have been wanting to add or resolve bugs or add features to make this project work better

**Comment by rochi88** [comment]
@LabibAhmedSiddique
Thanks and i appreciate your interest !!! 
Actually i also looking for contribution from other talented developers and help this library mature. You can add features of getting information like `p/e ratio`, `eps`,` 52 weeks' moving average` from below example page
"https://dsebd.org/displayCompany.php?name=ACI" 
if i'm not wrong the above information are crucial for decision making during trading.

Please feel free to provide any suggestion or pull request.
Thanks/Raisul
